.github/workflows/validate_package_schema.yml

```diff
@@ -1,22 +1,23 @@
-name: Validate Frictionless JSON Schemas for Spec
+name: Validate Frictionless Data Package Schema for Spec
 
-on: [push]
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Wouldn't it be helpful to validate for all PRs even if we don't require that it passes?

Reply:
This will validate for any PR that targets main, which should cover all cases for changes to the spec.

Reply:
Thinking through "the future" (which maybe we can change this for "the future" when that happens), wouldn't we want it to validate for PRs to any release-candidate type branches as well? I'm just thinking that there might not be a reason to specify a branch at all and if it gets messy/too much we can limit it to certain branches.

Reply:
I would be a little more worried about that if we were using the git-flow workflow, but we don't have release branches. I imagine releases will just be tags. The only other branch we would consider now is `ghpages`, but all but one of those files is built from main, and versioning the docs can be done through tags.

I guess maybe there's no big downside to removing the target branch restriction, without release branches it will function the same anyway.